web-local/src/lib/components/workspace/explore/filters/Filters.svelte

```diff
@@ -58,36 +58,51 @@ The main feature-set component for dashboard filters
   let searchText = "";
   let searchedValues = [];
   let activeDimensionName;
+  let hasTimeSeries;
+
+  $: if (metaQuery && $metaQuery.isSuccess && !$metaQuery.isRefetching) {
+    hasTimeSeries = hasDefinedTimeSeries($metaQuery.data);
```

Comment:
If you use the `useSelector()` pattern, then you probably don't need the if statement here